jena-arq/src/main/java/org/apache/jena/sparql/graph/NodeConst.java

```diff
@@ -58,9 +58,9 @@ private static Node uri(String uriStr) {
     public static final Node nodeANY       = Node.ANY;
 
 
-    public static final Node nodeOwlImports       = OWL.imports.asNode();   // uri("http://www.w3.org/2002/07/owl#imports")
-    public static final Node nodeOwlOntology      = OWL.Ontology.asNode();  // uri("http://www.w3.org/2002/07/owl#Ontology")
-    public static final Node nodeOwlSameAs        = OWL.sameAs.asNode();    // uri("http://www.w3.org/2002/07/owl#sameAs")
+    public static final Node nodeOwlImports       = OWL1.imports.asNode();   // uri("http://www.w3.org/2002/07/owl#imports")
+    public static final Node nodeOwlOntology      = OWL1.Ontology.asNode();  // uri("http://www.w3.org/2002/07/owl#Ontology")
+    public static final Node nodeOwlSameAs        = OWL1.sameAs.asNode();    // uri("http://www.w3.org/2002/07/owl#sameAs")
```

Comment:
Shouldn't these be `OWL` - they are OWL1/OWL2 neutral.
This happens quite a lot in this PR.
If that's the case, then OWL1 where it iued for constants can be converted to OWL as a separate later PR.